Contributing.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to ReadAlong Studio-Web
+
+Feel free to dive in! [Open an issue](https://github.com/ReadAlongs/Studio-Web/issues/new) or submit PRs.
+
+This repo follows the [Contributor Covenant](http://contributor-covenant.org/version/1/3/0/) Code of Conduct.
+
+This repo uses automated tools to standardize the formatting of code, text files and commits.
+
+- The [pre-commit hook](#pre-commit-hook) automatically applies code formatting rules using `prettier`.
+- [commitlint](#commitlint) is used as a commit message hook to validate that commit messages follow the convention.
```

Comment:
Looks good to me - I might make mention that this is a mono-repo managed by nx though. Stating that dependencies should be defined at the top level.

Reply:
and for information about spinning up development environments, consult the main readme.

Reply:
done